### PR TITLE
upgrade arkouda-helm-charts and arkouda-docker to v2023.05.05

### LINKED ISSUE #95: upgrade arkouda-helm-charts and arkouda-docker to v2023.05.05

### PR DESCRIPTION
**Pull request by hokiegeek2**
Closes #95 

This PR does the following:

1. upgrades arkouda-docker and arkouda-helm-charts to Arkouda v2023.05.05
2. fixes entrypoint error in arkouda-docker README.md
3. cleans up arkouda-helm-charts

### CODE REVIEW

**Review by Ethan-DeBandi99 — COMMENTED** [review]
Looks good overall. 

I wish there was a way within the README to use variables instead of having to update the version in a bunch of places. I did a quick search, but nothing that I found provided the proper formatting or worked within code blocks.

**Comment by hokiegeek2** [comment]
> Looks good overall.
> 
> I wish there was a way within the README to use variables instead of having to update the version in a bunch of places. I did a quick search, but nothing that I found provided the proper formatting or worked within code blocks.

@hokiegeek2 uses sed 🥇

File: arkouda-helm-charts/arkouda-udp-server/values.yaml [hunk 1]
```diff
@@ -11,53 +11,55 @@ resources:
...
     nodeport: # if service type is Nodeport
-    name: # service name
+    name: arkouda # service name
   metrics:
-    collectMetrics: # whether to collect metrics and make them available via  k8s service
+    collectMetrics: false # whether to collect metrics and make them available via  k8s service
     service:
-      name: # k8s service name for the Arkouda metrics service endpoint
+      name: arkouda-metrics # k8s service name for the Arkouda metrics service endpoint
       port: # k8s service port for the Arkouda metrics service endpoint, defaults to 5556
```
[the remark this answers is not in the document]
Comment (hokiegeek2) [reply arkouda-helm-charts/arkouda-udp-server/values.yaml:37]: @Ethan-DeBandi99 good point, I missed that. I need to update the deployment.yaml accordingly, good catch

**Review by jaketrookman — APPROVED** [review]
Looks good

File: arkouda-helm-charts/arkouda-udp-server/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hokiegeek2) [reply arkouda-helm-charts/arkouda-udp-server/values.yaml:36]: @Ethan-DeBandi99 updated deployment.yaml specify all default portst

**Review by pierce314159 — APPROVED** [review]

**Review by jaketrookman — APPROVED** [review]
Latest updates look good

**Review by Ethan-DeBandi99 — APPROVED** [review]

**Pull request merged**